snips_helper/intent.py

```diff
@@ -24,9 +24,27 @@ def utterances(self):
         self.__deactivate()
         return training_data
 
+    def import_utterances(self, utterances):
+        print("Importing utterances")
+        self.__activate()
+        self.__get_utterance_dot_button().click()
+        self.__get_import_utterance_button().click()
+        self._get_modal_text_area().send_keys(utterances)
+        self.__get_modal_import_button().click()
+        time.sleep(1)
+        self.save()
+        return
```

Comment:
we should deactivate after importing so the screen is backed out and further actions can be taken.